JavaScript/Topics/3.Web_Browser/README.md

```diff
@@ -1 +1,71 @@
-# To Do
\ No newline at end of file
+# Web Browser In Javascript
+
+Web Browsers are one of the the runtime for JavaScript, which means it's the environment where your JavaScript code gets executed also it's a platform that extends the core JavaScript and provides additional features through specific objects and functions.
+</br>
+* The global object for **JavaScript** in web browser is the window object. It represents web browser window and simultaneously the ECMAScript Global object. 
+
+* The window object has properties that specify details of the browser window **(navigator, location, history) and methods (alert(), setTimeot())** which perform various operations.
+
+* Another **important part of JavaScript in web browser** is handling events. Event handler allow you to define function, that is called when specified event (for example an click) occurs.
+
+* The most important of them is **XMLHttpRequest object** which enables communication with web-servers.
+
+## **In this tutorial, we will see How to write JavaScript and what are the methods to write it.**
+
+First give the title(of your choice) as "Web Browser in JavaScript Tutorial" under the \<title> tag.
+
+### **Code**
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Web Browser in JavaScript Tutorial</title>
+   
+</head>
+<body>
+    <div class="container">
+        <div class="row">
+            <p>
+              Girlscript Winter Of Contribution(GWOC)  
+            </p>
+        </div>
+    </div>
+    <script>
+        //Write your js here
+        console.log('Hello world');
+        
+    </script>
+</body>
+</html>
+
+```
+
+The result of the above code will look as follows on the HTML Page-
+
+<img title="a title" alt="Alt text" src="img1.png">
+
```

Comment:
Crop the image as the rest of the part is not required
Just show the tabs and the thing it is printing
Too much unnecessary white blank space of webpage is not needed